### DISCUSSION TITLE
Transparent background?

### DISCUSSION THREAD

**Discussion by michaelmoreno** [Textualize/textual#1234] [Q&A] [answered]
I want my terminal app to have a transparent background as to be able to see the desktop behind.
Is this supported?

**Comment by davep** [comment, marked as answer]
I would imagine this would be more of a terminal configuration thing than a Textual thing. Looks good here:

<img width="1210" alt="Screenshot 2022-11-20 at 08 12 41" src="https://user-images.githubusercontent.com/28237/202892225-76e17216-d0b2-4e9a-b415-fce137a4b46c.png">

You can also control how transparent the background is with the alpha portion of the colour:

<img width="1177" alt="Screenshot 2022-11-20 at 08 15 19" src="https://user-images.githubusercontent.com/28237/202892292-8442b2e8-f04c-47df-839d-cb72910f69c3.png">

**Reply by adrzhou** [reply]
Does this work for all terminal emulators? I'm using KDE Plasma on Kubuntu 22.04 and I configured Konsole to have 100% background transparency. But that setting does not take effect when I run a textual app.

![image](https://user-images.githubusercontent.com/101899715/213932635-e673aa75-d35e-42d5-a842-a1a97c86c51b.png)

**Reply by davep** [reply]
The effect, if it works, will depend on the terminal. The example I gave above is with the macOS terminal.app; on the other hand I get the same effect you're seeing here if I use iTerm2 (at least in the way I configured it).
